src/engine/input/NodeWithInput.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Godot;
+
+/// <summary>
+///   Input class for a <see cref="Godot.Node"/>
```

Comment:
Or maybe it would be better to have more comprehensive documentation here, and then link to this from ControlWithInput.